optaplanner-wb-screens/optaplanner-wb-solver-editor/optaplanner-wb-solver-editor-backend/src/test/java/org/optaplanner/workbench/screens/solver/backend/server/TestUtil.java

```diff
@@ -20,8 +20,16 @@
 import java.io.InputStreamReader;
 import java.io.Reader;
 
+import javax.enterprise.inject.Produces;
+
+import org.appformer.project.datamodel.commons.util.MVELEvaluator;
+import org.appformer.project.datamodel.commons.util.RawMVELEvaluator;
+
 public class TestUtil {
 
+    @Produces
```

Comment:
Is the evaluator required? Just can't see it being injected anywhere around, could you elaborate? Thanks.

Reply:
@mcimbora Yes the evaluator is injected in several places in kie-wb-common (where as it used to be created via static methods). Without this Weld gets an exception causing tests in this module to fail.

In the KIE webapps the producer is supplied by `drools-cdi`, so now any tests using Weld and the DMO need to either depend on `drools-cdi` or provide their own evaluator.